spyderlib/plugins/editor.py

```diff
@@ -942,13 +975,22 @@ def get_plugin_actions(self):
         self.recent_file_menu = QMenu(_("Open &recent"), self)
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 
-        file_menu_actions = [self.new_action, self.open_action,
-                             self.recent_file_menu, self.save_action,
+        file_menu_actions = [self.new_action, 
+                             None,
+                             self.open_action,
+                             self.recent_file_menu,
+                             None,
+                             None,

```

Comment:
I'd also to see a new `Projects` menu entry, so that's really visible to our users.